core/src/validator.rs

```diff
@@ -299,19 +299,22 @@ pub struct ValidatorConfig {
     pub repair_handler_type: RepairHandlerType,
 }
 
-impl Default for ValidatorConfig {
-    fn default() -> Self {
+impl ValidatorConfig {
+    pub fn default_for_test() -> Self {
```

Comment:
@t-nelson - Continuing the conversation from #7100, I set things up to remove `Default` completely and instead leave the `default_for_test()` method. To your question, `default_for_test()` cannot yet be DCOU; there are several places that need to be updated. One of those being `local-cluster`.

I started the convo [here](https://discord.com/channels/428295358100013066/439194979856809985/1397075687587971122) and will probably make the PR to do that this afternoon